ci-operator/step-registry/openshift/e2e/test/hypershift-qe/mgmt/longduration/openshift-e2e-test-hypershift-qe-mgmt-longduration-chain.yaml

```diff
@@ -0,0 +1,8 @@
+chain:
+  as: openshift-e2e-test-hypershift-qe-mgmt-longduration
+  steps:
+  - ref: cucushift-hypershift-extended-health-check
+  - ref: idp-htpasswd
+  - ref: openshift-extended-test-longduration
+  documentation: |-
+    Execute e2e tests from QE, which include golang (openshift-extended-test), cucushift (cucushift-e2e), ...(more to add)
```

Comment:
The documentation needs update.

Reply:
@liangxia Thanks a lot, Could you help to review again?